Source/Fuse.Reactive.Bindings/Let.uno

```diff
@@ -0,0 +1,279 @@
+using Uno;
+using Uno.Collections;
+using Uno.UX;
+
+namespace Fuse.Reactive
+{
+	/**
+		Binds an expression or value to a name in the data context. This simplifies repeated calculations and allows introduction of new variables.
+		
+		To introduce a new value:
+		
+			<Let Value="5" ux:Name="a"/>
+			
+		The value is now part of the data context:
+		
+			<Slider Value="{a}"/>
+		
+		It can also be accessed directly outside of the context:
+		
+			<Slider Value="{Property a.Value}"/>
+			
+		
+		If you are using an expression it's recommended now to use `Expression` instead of value:
+		
+			<Let Expression="{pos} + 5" ux:Name="p"/>
```

Comment:
should we have ux:Name as the first attribute in these examples? as per the normal convention